ethcore/src/miner/pool_client.rs

```diff
@@ -36,10 +36,7 @@ use header::Header;
 use miner;
 use miner::service_transaction_checker::ServiceTransactionChecker;
 
-type NoncesCache = RwLock<HashMap<Address, U256>>;
-
-const MAX_NONCE_CACHE_SIZE: usize = 4096;
-const EXPECTED_NONCE_CACHE_SIZE: usize = 2048;
+type NoncesCache = (RwLock<HashMap<Address, U256>>, usize);
```

Comment:
We should probably create a struct for this at some point.

Reply:
Done